core/Cargo.toml

```diff
@@ -16,8 +16,8 @@ serde = ["dep:serde", "mc-core-types/serde"]
 default = ["bip39", "mc-crypto-dalek/default"]
 
 [dependencies]
-curve25519-dalek = { version = "4.0.0-pre.2", default-features = false }
-ed25519-dalek = { version = "2.0.0-pre.1", default-features = false }
+curve25519-dalek = { version = "4.0.0-rc.1", default-features = false }
+ed25519-dalek = { version = "2.0.0-pre.0", default-features = false }
```

Comment:
Is ed25519-dalek downgrading from pre.1 to pre.0 deliberate here? I haven't been following the curve25519 changes so it's quite likely I'm just missing context.

Reply:
We're stopping the use of our own forks of these projects---c.f. the top-level and `enclave/trusted/` variants of `Cargo.toml`.

Many years ago, I'd advanced the version of ed25519-dalek to be later than the released version.